cns/ipamclient/ipamclient_test.go

```diff
@@ -14,13 +14,14 @@ import (
 	"github.com/Azure/azure-container-networking/common"
 )
 
-var mux *http.ServeMux
-var ipamQueryUrl = "localhost:42424"
-var ic *IpamClient
+var (
+	mux          *http.ServeMux
```

Comment:
Looks like "mux" is not used, can we check with whoever added if this can be removed ?